### ISSUE TITLE
Convert project to TypeScript

### ISSUE THREAD

**Issue by paceaux** [paceaux/methodius#18]
Project should be in TS b/c that would be easier for documentation.

**Comment by paceaux** [comment]
How to test this

Create a new project. install Methodius. use `npm link`. 

use this code:

```

const Methodius = require('methodius');
const test = new Methodius('Hey there frank. How is it going? Is there anything you would like to do? I am wondering what happens here. I am excited. This should be fun.');


const ngrams = test.bigrams;

console.log(ngrams);
console.log(test.trigrams);
```

It should run.

**Comment by paceaux** [comment]
This runs without error. It was a lot of effort. Got released in 2.0.0 last night.  Someone please clap. 

:clap:
